tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/0-get-started/TC_05-sitetype-primary.cy.js

```diff
@@ -0,0 +1,109 @@
+// <reference types="Cypress" />
+import { CheckDrawerDisabled } from '../../wp-module-support/drawer.cy';
```

Comment:
Should we `camel` case function names like `CheckDrawerDisabled`, `CheckCardHeadingSubheading` etc?